vmdk_plugin/plugin.go

```diff
@@ -209,6 +221,10 @@ func (d *vmdkDriver) unmountVolume(name string) error {
 // Name and driver specific options passed through to the ESX host
 func (d *vmdkDriver) Create(r volume.Request) volume.Response {
 
+	// Lock this volume for create
+	d.lockVolume(r.Name)
+	defer d.unlockVolume(r.Name)
```

Comment:
After having a second thought, without locking on create we can race while formating the filesystem with the volume attached and another container tries to start with that volume. So locking here is really needed.